components/button/button-subtle.js

```diff
@@ -125,18 +127,19 @@ class ButtonSubtle extends ButtonMixin(RtlMixin(LitElement)) {
 				:host([active]:not([disabled])) button .d2l-button-subtle-content {
 					color: var(--d2l-color-celestine-minus-1);
 				}
-				:host([icon]) .d2l-button-subtle-content,
-				:host([dir="rtl"][icon][icon-right]) .d2l-button-subtle-content {
+				:host([_has-icon]) .d2l-button-subtle-content,
+				:host([dir="rtl"][_has-icon][icon-right]) .d2l-button-subtle-content {
 					padding-left: 1.2rem;
 					padding-right: 0;
```

Comment:
We've been slowly adopting CSS logical properties on new things... since they would simplify these changes (and possibly remove the need for `RtlMixin`) it could be a good time to do that here. Thoughts?

Reply:
The details in the note there says that Safari <15 only doesn't support the shorthand ones when the value is a CSS property. SO in this particular case, it would be fine. Safari 14 is 3 years old at this point as well.